rocketpy/Flight.py

```diff
@@ -2253,12 +2212,103 @@ def allInfo(self):
+                sum(self.functionEvaluationsPerTimeStep) / len(self.timeSteps))
         )
 
-        # Kinematics plots
-        print("\n\nTrajectory Kinematic Plots\n")
+        return None
+
+    def maxWindVelocityPrints(self, maxAngle):
+        """ Function to calculate the maximum wind velocity before the angle of
+        attack exceeds a desired angle, at the instant of departing rail launch.
+        Can be helpful if you know the exact stall angle of all aerodynamics 
+        surfaces. 
+
+        Parameters
+        ----------
+        maxAngle : float
+            Angle, in degrees, for which you would like to know the maximum wind
+            speed before the angle of attack exceeds it
+        Return
+        ------
+        None
+        """
+        vF = self.outOfRailVelocity
+
+        # Convert angle to radians
+        tetha = self.inclination * np.pi /180
+        maxAngle = maxAngle * np.pi /180
+
+        c = (math.cos(maxAngle)**2 - math.cos(tetha)**2)/ math.sin(maxAngle)**2
+        wV = (2*vF*math.cos(tetha)/c + (4*vF*vF*math.cos(tetha)*math.cos(tetha)/(c**2) + 4*1*vF*vF/c )**0.5 )/2
+        
+        # Convert maxAngle to degrees
+        maxAngle = maxAngle * 180 / np.pi
+        print("Maximum wind velocity at Rail Departure time before angle of attack exceeds {:.3f}°: {:.3f} m/s".format(maxAngle, wV))
+
+        return None
+
+    def trajectory3dPlot(self):
+        """Plot a 3D graph of the trajectory
+
+        Parameters
+        ----------
+        None
+        
+        Return
+        ------
+        None
+        """
+        # Post-process results
+        if self.postProcessed is False:
+            self.postProcess()
+
+        # Get max and min x and y
+        maxZ = max(self.z[:, 1] - self.env.elevation )
+        maxX = max(self.x[:, 1])
+        minX = min(self.x[:, 1])
+        maxY = max(self.y[:, 1])
+        minY = min(self.y[:, 1])
+        maxXY = max(maxX, maxY)
+        minXY = min(minX, minY)
+
+        # Create figure
+        fig1 = plt.figure(figsize=(9, 9))
+        ax1 = plt.subplot(111, projection="3d")
+        ax1.plot(
+            self.x[:, 1], self.y[:, 1], zs= 0, zdir="z", linestyle="--"
+        )
+        ax1.plot(self.x[:, 1], self.z[:, 1] - self.env.elevation, zs=minXY, zdir="y", linestyle="--")
+        ax1.plot(self.y[:, 1], self.z[:, 1] - self.env.elevation, zs=minXY, zdir="x", linestyle="--")
+        ax1.plot(self.x[:, 1], self.y[:, 1], self.z[:, 1] - self.env.elevation, linewidth='2')
+        ax1.scatter(0, 0, 0)
+        ax1.set_xlabel("X - East (m)")
+        ax1.set_ylabel("Y - North (m)")
+        ax1.set_zlabel("Z - Altitude Above Ground Level (m)")
+        ax1.set_title("Flight Trajectory")
+        ax1.set_zlim3d([0, maxZ])
+        ax1.set_ylim3d([minXY, maxXY])
+        ax1.set_xlim3d([minXY, maxXY])
+        ax1.view_init(15, 45)
+        plt.show()
+
+        return None
+
+    def kinematicsPlots(self):
```

Comment:
The code seems to be printing the 3D plot again... is this intended behavior?

Reply:
No, it is not. I will fix this problem!